app/models/comment.rb

```diff
@@ -2,4 +2,8 @@ class Comment < ApplicationRecord
   include Visible
 
   belongs_to :article
+
+  validates :commenter, presence: true
+  validates :body, presence: true
+  validates :article_id, presence: true
```

Comment:
You don't need to validate presence of foreign keys. Try to experiment with it :)